test/util/namespaces/namespaces.go

```diff
@@ -46,29 +46,35 @@ func Create(namespace string, cs *testclient.ClientSet) error {
 	return err
 }
 
-// Clean cleans all dangling objects from the given namespace.
-func Clean(operatorNamespace, namespace string, cs *testclient.ClientSet) error {
+func namespaceExists(namespace string, cs *testclient.ClientSet) bool {
 	_, err := cs.Namespaces().Get(namespace, metav1.GetOptions{})
-	if err != nil && k8serrors.IsNotFound(err) {
+	return err == nil || k8serrors.IsNotFound(err)
```

Comment:
When k8serrors.IsNotFound(err) this should return false, because the namespace does not exists

Reply:
Done